pkg/ovs/ovsctl/interface.go

```diff
@@ -31,9 +31,25 @@ type TracingRequest struct {
 	AllowOverrideInPort bool
 }
 
-// OVSCtlClient is an interface for executing OVS "ovs-ofctl" and "ovs-appctl"
-// commands.
+type OVSOfctlRunner interface {
+	RunOfctlCmd(cmd string, args ...string) ([]byte, error)
+}
+
+type ovsOfctlRunner struct {
```

Comment:
the concrete types (`ovsOfctlRunner` and `ovsAppctlRunner`) should not be in `interface.go`, which is reserved for interfaces